frontend/apps/web/yup-validations/jobs.ts

```diff
@@ -274,3 +274,11 @@ export const DataSyncSourceFormValues = SourceFormValues.concat(
 export type DataSyncSourceFormValues = Yup.InferType<
   typeof DataSyncSourceFormValues
 >;
+
+export const DEFAULT_TRANSFORMER_FORM_VALUES = Yup.object({
```

Comment:
Moving away from having this and the type be different. This value should be the same casing as the type so they are intertwined. 
```suggestion
export const DefaultTransformerFormValues = Yup.object({
```